.github/workflows/ci.yml

```
@@ -1,13 +1,24 @@
-name: ci
+﻿name: ci
 
 on:
   push:
-    branches: [ main, master ]
+    branches: [ main, develop, master, feature/** ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)